### ISSUE TITLE
Error: page.evaluate: Evaluation failed: Error (Crunchyroll)

### ISSUE THREAD

**Issue by einkoro** [animeloyalty/animesync#27]
When trying to download Blade Runner Black Out 2022 from Canada I am consistently encountering the following error. So far other series have been downloading fine.
```
Error: page.evaluate: Evaluation failed: Error
    at mapSource (eval at callFunction (:307:29), <anonymous>:28:15)
    at evaluateStream (eval at callFunction (:307:29), <anonymous>:3:21)
    at UtilityScript.callFunction (<anonymous>:308:24)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at Connection.sendMessageToServer (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/connection.js:69:15)
    at Proxy.<anonymous> (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/channelOwner.js:44:61)
    at /Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/frame.js:157:48
    at Frame._wrapApiCall (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/channelOwner.js:72:34)
    at Frame.evaluate (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/frame.js:156:21)
    at /Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/page.js:344:60
    at Page._attributeToPage (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/page.js:205:20)
    at Page.evaluate (/Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/node_modules/playwright-core/lib/client/page.js:344:21)
    at /Users/einkoro/.nvm/versions/node/v14.15.2/lib/node_modules/animesync/dist/remote/services/crunchyroll/Crunchyroll.js:103:39
    at runNextTicks (internal/process/task_queues.js:58:5)
```

Are you able to reproduce it on your end?  Let me know if you need the full log.

**Comment by Deathspike** [comment]
It turns out that *Blade Runner Black Out 2022* is a premium video in Canada.
Did you log in *animesync* into your account with valid subscription?
See https://github.com/animeloyalty/animesync#crunchyroll
Alternatively, other regions like USA and regions of Europe, don't require a subscription for the movie..

**Comment by einkoro** [comment]
I did login with a premium subscription and it is watchable in app and on the website for me. The only thing worth noting there was I had to use the API to set my credentials as the CLI didn't seem to be saving them for both CR and Funimation.

**Comment by Deathspike** [comment]
Thanks for the additional information!
Unfortunately I'll have to delay fixing this one for a bit.
It sounds quite silly, but I don't have a premium account right now.
I'll get back to this once I have a subscribed account again :-\

**Comment by einkoro** [comment]
I'm in no particularly rush but I like the project and very much appreciate the normalize-ssa package. Check your inbox for a membership 🎁 ;)

**Comment by Deathspike** [comment]
Sorry for the late response. Got a bit side tracked. Thanks a bunch for the membership gift! I'm happy you're enjoying both animesync and normalize-ssa 😁

I know you're in no hurry, but I'll tackle this one in the coming days. Thanks again 👍

**Comment by Deathspike** [comment]
Fixed and released in `0.11.7`.
The problem was that Crunchyroll stream code was filtering non-Japanese streams. That code should not be necessary anymore, since dubs are filtered on a series-basis nowadays. But for English-only entries, like this movie, that code was problematic.
Let me know if this works for you now 👍

**Comment by einkoro** [comment]
Confirmed fixed for me with 0.11.7
